### PR TITLE
Dylan-Verwimp

### PR DESCRIPTION
**Pull request by Avadakedavr0**

### CODE REVIEW

File: src/sections/Advice/index.js [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
+  }, [])
+
   return (
     <section>
       <h2>Advice Section</h2>
-      <section className="adivce-slip"></section>
-      <section className="favourtite-slips-list"></section>
+      <AdviceSlip advice={advice} getMoreAdvice={fetchAdvice} addToFavorites={addToFavorites} />
+      <FavoriteSlipsList favorites={favorites} />
+      {favorites.length > 0 && (
```
Comment (vherus) [inline src/sections/Advice/index.js:35]: `.length` is a truthy value, so you could remove the comparison and just do `favourites.length &&`

Comment (Avadakedavr0) [reply src/sections/Advice/index.js:35]: Amazing, thank you for telling me! 
A small question, what if favourites.length would be 0? Then we put out a 0?

Comment (vherus) [reply src/sections/Advice/index.js:35]: If it's 0, it'll be treated as false. So by doing `{something.length && <Component />}`, the component will render if length is 1 or above, and won't render if it's 0

Comment (Avadakedavr0) [reply src/sections/Advice/index.js:35]: Great, thank you!

**Pull request closed without merging**